File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-2.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */

class Solution {
    public List<List<Integer>> levelOrder(TreeNode root) {
        List<List<Integer>> L = new ArrayList<>() ;
        if(root == null) return L ;  
        Queue<TreeNode> queue = new LinkedList<>() ; 

        queue.offer(root) ; 

        while(!queue.isEmpty())
        {
            int size = queue.size() ;
            List<Integer> insideList = new ArrayList<>(); 
            for(;size-- >0 ;)
            {  TreeNode value = queue.remove() ;

               insideList.add(value.val); 

               if(value.left != null)  queue.offer(value.left);

               if(value.right != null) queue.offer(value.right);
              

            }
            L.add(insideList);
        }

        return L ; 


        
    }
}

File: Data Structures & Algorithms/valid-binary-search-tree/submission-2.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def isValidBST(self, root: Optional[TreeNode]) -> bool:
        the_list : List[int] = self.myList(root)
        length : int = len(the_list)
        for i in range(1,length):
            if the_list[i] <= the_list[i-1]: return False
        return True



    def myList(self,root:Optional[TreeNode]) -> List[int]:
        if root is None : return []

        return self.myList(root.left)+[root.val]+self.myList(root.right)    
        